azkaban-common/src/test/java/azkaban/utils/DependencyTransferManagerTest.java

```diff
@@ -74,6 +78,7 @@ public void setup() throws Exception {
 
     Props sampleProps = new Props();
     sampleProps.put(Constants.ConfigurationKeys.AZKABAN_DEPENDENCY_MAX_DOWNLOAD_TRIES, DEPENDENCY_DOWNLOAD_MAX_TRIES);
+    sampleProps.put(AZKABAN_DEPENDENCY_DOWNLOAD_TIMEOUT_SECONDS, 10);
```

Comment:
suggest we lower to 2s, so as less test overhead

Reply:
I retried that, unfortunately the other tests in this class would randomly fail. We need some number that is good enough to cause less flakyness. I will try if 5s works

Reply:
change to 3s.